haystack/document_store/elasticsearch.py

```diff
@@ -44,7 +44,7 @@ def __init__(
         refresh_type: str = "wait_for",
         similarity="dot_product",
         timeout=30,
-        return_embedding: Optional[bool] = True,
+        return_embedding: bool = True,
```

Comment:
Wouldn't it be better to make the default `False` here? The embeddings can become quite heavy and in many cases we don't actually need them (e.g. in a DPR -> Reader pipeline)

Reply:
(similarly for the other document_stores)